internal/daemon/controller/auth/testing.go

```diff
@@ -20,7 +21,7 @@ func DisabledAuthTestContext(iamRepoFn common.IamRepoFactory, scopeId string, op
 	}
 	reqInfo.UserIdOverride = opts.withUserId
 	if reqInfo.UserIdOverride == "" {
-		reqInfo.UserIdOverride = "u_auth"
+		reqInfo.UserIdOverride = perms.AdminUserId
```

Comment:
This doesn't seem right. There's no such thing as an "admin user", at least in terms of built-ins.